src/acl/AdaptationService.h

```diff
@@ -9,15 +9,21 @@
 #ifndef SQUID_ACLADAPTATIONSERVICE_H
 #define SQUID_ACLADAPTATIONSERVICE_H
 
-#include "acl/Strategy.h"
+#include "acl/Data.h"
+#include "acl/ParameterizedNode.h"
 
-/// \ingroup ACLAPI
-class ACLAdaptationServiceStrategy : public ACLStrategy<const char *>
+namespace Acl
 {
 
+/// an "adaptation_service" ACL
+class AdaptationServiceCheck: public ParameterizedNode< ACLData<const char *> >
```

Comment:
I did not rename this and other old source files even though we are changing the primary class names in most of them. We should eventually rename (and perhaps even merge trivial .cc and .h files in most cases), but I believe such renaming deserves a _dedicated_ PR to avoid creating a lot of out-of-scope noise here. This PR did not create this old problem: The old file names were often inconsistent with primary class names (e.g., this file was no called AdaptationServiceStrategy.h or similar). Moreover, some class names were (and remain) rather problematic (e.g. official code has both ACLRequestHeaderStrategy and ACLHTTPReqHeaderStrategy that became RequestHeaderCheck and HttpReqHeaderCheck).